tests/phpunit/tests/image/intermediateSize.php

```diff
@@ -93,7 +93,7 @@ static function() {
 	 * @requires function imagejpeg
 	 */
 	public function test_get_intermediate_sizes_by_name() {
-		add_image_size( 'test-size', 330, 220, true );
+		add_image_size( 'test-size', 330, 220, true, true );
```

Comment:
Why pass the new parameter here (and throughout this file)? This seems pointless, we shouldn't need to pass it if it's optional and not relevant to the specific test.